dp/generate_climos.py

```diff
@@ -134,19 +134,34 @@ def climo_outputs(time_resolution):
     # Apply per-variable processing
     # - Scale pr variable if it is not in desired units
     #   - TODO: Use UDUNITS here for more robust processing of units? It has a fairly heavy API. But this is uuuugly.
+    pr_attributes = {}  # will contain updates, if any, to pr variable attributes
     if 'pr' in input_file.dependent_varnames:
-        units = input_file.variables['pr'].units
+        pr_variable = input_file.variables['pr']
+        units = pr_variable.units
         if any(u in units for u in ['kg', 'mm']) and any(u in units for u in ['/s', '/ s', 's-1', 's^-1']):
             logger.info("Converting 'pr' variable to units mm/day")
-            # Extract variable
-            pr_only = cdo.select('name=pr', input=climo_means)
-            # Multiply values by 86400 to convert from mm/s to mm/day
-            pr_only = cdo.mulc('86400', input=pr_only)
-            # Replace pr in all-variables file
-            climo_means = cdo.replace(input=[climo_means, pr_only])
             # Update units attribute
-            # TODO: Verify that "d-1" is desired way to express "per day" (alternaive is "day-1")
-            pr_units_attr = re.sub('(/s|/ s|s-1|s\^-1)', ' d-1', units)
+            pr_attributes['units'] = re.sub('(/s|/ s|s-1|s\^-1)', ' d-1', units)
```

Comment:
I've seen s-1 for "per second", but I've never seen d-1 for "per day". Is that in the SI units?